src/DbPostgreSQLAdapter/Field.cpp

```diff
@@ -0,0 +1,185 @@
+#include "stdafx.h"
+#include "Field.h"
+
+namespace {
+	std::chrono::system_clock::time_point stringISOToDateTime(const std::string& dateTime)
+	{
+		std::chrono::system_clock::time_point timePointDateTime;
+		std::istringstream{ dateTime } >> std::chrono::parse("%F %T%z", timePointDateTime);
```

Comment:
With space separating date and time or with a _T_ in the middle?

Reply:
with space separating it. Curiously, Postgres does not support a date time format with the T in the middle (another weirdness of this engine).